scripts/build.d/xcb

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+
```

Comment:
@terrychan999 Could you add a section of comments to summarize the packages included in this script?

Reply:
Thanks.  In the comment block please describe why keeping the 9 packages in a script is a good idea.  Then I am good with it.